launchable/test_runners/nunit.py

```diff
@@ -0,0 +1,77 @@
+import glob
+import os
+
+import click
+
+from launchable.commands.record.case_event import CaseEvent
+from launchable.utils.sax import SaxParser, Element
+from . import launchable
+from launchable.testpath import TestPath
+
+# common code between 'subset' & 'record tests' to build up test path from nested <test-suite>s
+def build_path(e: Element):
+    pp = [] # type: TestPath
+    if e.parent:
+        pp = e.parent.tags.get('path') or []    # type: ignore
+    if e.name == "test-suite":
+        # <test-suite>s form a nested tree structure so capture those in path
+        e.tags['path'] = pp + [{'type': e.attrs['type'], 'name': e.attrs['name']}]
```

Comment:
TIL:  add elements to array use by `+` operator in Python

Reply:
Because Python allows operator overloading. Especially Numpy uses this feature a lot.